doc/cephfs/fs-volumes.rst

```diff
@@ -1340,5 +1340,28 @@ set with this id was present in the database
 
   $ ceph fs quiesce fs1 sub1 sub2 sub3 --set-id="external-id" --if-version=0
 
+
+.. _disabling-volumes-plugin:
+
+Disabling Volumes Plugin
+------------------------
+By default, on a Ceph cluster, volumes plugin is enabled as well as set as
```

Comment:
By default the ``volumes`` plugin is enabled and set to ``always on``.

Reply:
Done.